dt-reports/magic-url-class.php

```diff
@@ -547,7 +549,13 @@ public function api_create_key( WP_REST_Request $request ){
          * Filters and returns registered types that allow bulk send.
          * @return array
          */
-        public static function list_bulk_send() {
+        public static function list_bulk_send( $post_type = null ) {
+
+            /**
+             * Ensure to pre-load any available magic link templates prior to
+             * main list_bulk_send() execution.
+             */
+            do_action( 'dt_magic_link_load_templates', $post_type );
```

Comment:
what if we hooked in to `dt_magic_url_register_types` instead 
Since that is what is called in the next line?

Reply:
Spawns a recursive loop... Approach to be discussed further..